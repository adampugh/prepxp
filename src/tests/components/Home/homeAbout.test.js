import React from "react";
import { shallow } from "enzyme";

import HomeAbout from "../../../components/Home/homeAbout";

describe("<HomeAbout />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<HomeAbout />);
        expect(wrapper).toMatchSnapshot();
    });
});