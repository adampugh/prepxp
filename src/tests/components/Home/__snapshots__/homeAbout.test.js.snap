// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeAbout /> should match snapshot 1`] = `
<main>
  <div
    className="about__title"
  >
    <h1>
      Why use PrepXP in your job search?
    </h1>
  </div>
  <div
    className="about__grid"
  >
    <div>
      <img
        alt="arm icon"
        className="bannerText__image"
        src="interview.png"
      />
    </div>
    <div
      className="about__grid__text"
    >
      <h1>
        Need help with upcoming interviews?
      </h1>
      <p>
        PrepXP aids users in landing their dream jobs by helping them pass the interview. Checkout our curated lists based on questions asked by some of the largest companies in the world.
      </p>
      <Link
        replace={false}
        to="/login"
      >
        <button
          className="btn"
        >
          Start now
        </button>
      </Link>
    </div>
  </div>
</main>
`;
